packages/api-form-builder/src/plugins/crud/forms.crud.ts

```diff
@@ -391,7 +340,7 @@ export const createFormsCrud = (params: CreateFormsCrudParams): FormsCRUD => {
             await updateData.validate();
             const data = await updateData.toJSON({ onlyDirty: true });
```

Comment:
@neatbyte-ivelychko Let's tweak this a bit. Set `enabled` to have a default value of `false`. `siteKey` and `secretKey` are irrelevant if `enabled: false`. With that, you can update the test to only expect the `enabled` key to be set.

Reply:
Okay, I will do test with `enabled: false`.

Reply:
You can use the `.default(false)` to set default values right there on the schema: https://zod.dev/?id=default

Reply:
I have fixed this issue with validation by adding a `.deepPartial()` to the `Zod` models. After doing that `formInstallion.test.ts` started working again, but we have started getting the `reCaptcha: null` responce in the `settings.test.ts` file instead of getting this expected responce:

```
reCaptcha: {
   enabled: null,
   siteKey: null,
   secretKey: null
}
```

And I want to know whether it is okay, because we need to change expected responces in the `settings.test.ts` file. 
And in the `API` we would change `Settings` type in `api-form-builder` because `values` from `reCaptcha` object could be `undefined` after a `.deepPartial()` change in the model.

@leopuleo @Pavel910

Reply:
@neatbyte-ivelychko you know what.... let's do this in a safer way. Let's keep the data as it was before, to reduce the risk of breaking stuff on the UI side. So for now, just add default values into the zod schema for all 3 attributes. maybe you'll also need to set a default value for the whole `reCaptcha` attribute. But in any case, let's not use `deepPartial` because that basically allows anything, it's almost like `any` in typescript. I would rather have `null` but have actual object structure respected.